tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CalendarEventReplyPerformer.scala

```diff
@@ -144,26 +160,44 @@ class CalendarEventMailReplyGenerator(val bodyPartContentGenerator: CalendarRepl
   private def generateMailName(): String = "calendar-reply-" + UUID.randomUUID().toString
 }
 
-case class ReplyBodyPart(subject: String, bodyPart: BodyPartBuilder)
+case class ReplyBodyPart(subject: String, bodyPart: BodyPart)
 
 trait CalendarReplyBodyPartContentGenerator {
   def getBodyPartContent(i18n: Locale, attendeeReply: AttendeeReply, calendar: Calendar): SMono[ReplyBodyPart]
 }
 
-class FakeMailCalendarEventReplyContent extends CalendarReplyBodyPartContentGenerator {
+class I18NMailCalendarEventReplyContent(fileSystem: FileSystem, i18nEmlDirectory: String) extends CalendarReplyBodyPartContentGenerator {
 
   val dateTimeFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("EEE MMM dd, yyyy")
+  private val ATTENDEE_TAG: String = "{{ATTENDEE}}"
 
   override def getBodyPartContent(i18n: Locale, attendeeReply: AttendeeReply, calendar: Calendar): SMono[ReplyBodyPart] = {
-    SMono.just(attendeeReply.attendee.asMailAddress().toString + " has " + attendeeReply.partStat.getValue.toLowerCase + " this invitation")
-      .map(MimeMessageBuilder.bodyPartBuilder.data(_))
-      .map(bodyPart => ReplyBodyPart(generateSubject(attendeeReply, calendar), bodyPart))
+    val emlFilename: String = evaluateMailTemplateFileName(attendeeReply.partStat, i18n)
+    val emlLocation: String = URI.create(i18nEmlDirectory).resolve(emlFilename).toString
+
+    SMono.fromCallable(() => fileSystem.getResource(emlLocation))
+      .map(IOUtils.toByteArray)
+      .map(MimeMessageUtil.mimeMessageFromBytes)
+      .map(mimeMessage => {
+        val bodyPart = mimeMessage.getContent match {
+          case textBody: String => MimeMessageBuilder.bodyPartBuilder.data(decorateEmailBody(textBody, attendeeReply)).build()
+          case _ => throw new IllegalArgumentException("The eml file must contain a text body.")
```

Comment:
I'm puzzled about how to handle this when the eml template file has the content format not text (maybe Multipart...)